### PR TITLE
304: Read Me

### PR DESCRIPTION
**Pull request by halprin**
## What changed

Updated the `README.md` based on all the changes we've made.  I also slimmed it down a little bit.

## Issue

#304.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,177 +1,204 @@
...
-Technologies used in this prototype:
-* Python
-* Django 4.x
-* React
-* PostgreSQL
-* Cloud.gov
+## Dependencies
 
-## Getting Started
+At a bare minimum, you need [Docker](https://www.docker.com) installed to run the application locally.  If you want to do
```
Comment (tdonaworth) [inline README.md:11]: I don't recall, but does `Docker` include `Compose` now? I know it didn't in the past, and it was a separate install on some systems.

Comment (halprin) [reply README.md:11]: Oh, good point.  I know it does with Docker Desktop for ____.  Let me check.

Comment (halprin) [reply README.md]: I added a bit that we also require Docker Compose.  The webpage linked to mentions that you might already have it through Docker Desktop and also includes instructions on how to install it if you just have the engine/CLI installed.

Comment (tdonaworth) [reply README.md]: 👍

**Review by tdonaworth — APPROVED** [review]
Nice cleanup!

**Pull request merged**